app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt

```diff
@@ -94,5 +91,8 @@ class MainFragment : Fragment() {
             }
         }
     }
+    companion object {
```

Comment:
moved the companion object because if you look at the naming conventions document (see link), you can see that the class layout is:
1) Property declarations and initializer blocks
2) Secondary constructors
3) Method declarations
4) Companion object

and you had the companion object at the top of your document when the convention says it should be at the bottom.
Overall everything else looks good so far, not a lot of code but looking forward to seeing the changes that are going to be made.